sympy/conftest.py

```diff
@@ -23,10 +30,11 @@ def process_split(config, items):
         return
     m = sp.match(split)
     if not m:
-        raise ValueError("split must be a string of the form a/b "
-                         "where a and b are ints.")
+        raise ValueError(
+            "split must be a string of the form a/b " "where a and b are ints."
```

Comment:
If you decide to keep the style changes intact here, at least fix this so it's just one string.